requirements.txt

```diff
@@ -1,73 +1,15 @@
-alembic==1.4.2
-aniso8601==3.0.0
-appdirs==1.4.3
-APScheduler==3.5.1
-asn1crypto==0.24.0
-attrs==19.3.0
-awsebcli==3.14.1
-bcrypt==3.1.7
-black==19.3b0
-blessed==1.15.0
+APScheduler==3.6.3
 blinker==1.4
-botocore==1.10.48
-cached-property==1.4.3
-cement==2.8.2
-certifi==2018.4.16
-chardet==3.0.4
-click==6.7
-colorama==0.3.9
-coverage==4.5.1
-cryptography==2.8
-docker==3.4.0
-docker-compose==1.21.2
-docker-pycreds==0.3.0
-dockerpty==0.4.1
-docopt==0.6.2
-docutils==0.14
-entrypoints==0.3
+coverage==5.3
 Flask==1.0.2
-Flask-JWT-Extended==3.10.0
+Flask-JWT-Extended==3.24.1
 Flask-Mail==0.9.1
-Flask-Migrate==2.5.2
+Flask-Migrate==2.5.3
 flask-restx==0.2.0
-Flask-SQLAlchemy==2.3.2
-Flask-Testing==0.7.1
-future==0.16.0
+Flask-Testing==0.8.0
 gunicorn==20.0.4
-idna==2.6
-itsdangerous==0.24
-Jinja2==2.10
-jmespath==0.9.3
-jsonschema==2.6.0
-Mako==1.1.3
-MarkupSafe==1.1.1
-mccabe==0.6.1
-paramiko==2.7.1
-pathspec==0.5.5
-psycopg2-binary==2.8.5
-pycodestyle==2.5.0
-pycparser==2.18
-pyflakes==2.1.1
-PyJWT==1.4.2
-PyMySQL==0.9.2
-PyNaCl==1.3.0
-python-dateutil==2.7.3
-python-dotenv==0.8.2
-python-editor==1.0.4
-pytz==2018.4
-PyYAML==3.13
-regex==2020.2.20
-requests==2.18.4
-semantic-version==2.5.0
-six==1.11.0
-SQLAlchemy==1.2.7
-tabulate==0.7.5
-termcolor==1.1.0
-texttable==0.9.1
-toml==0.10.0
-typed-ast==1.4.1
-tzlocal==1.5.1
-urllib3==1.22
-wcwidth==0.1.7
-websocket-client==0.48.0
-Werkzeug==0.16.1
+psycopg2-binary==2.8.6
+PyMySQL==0.10.1
+python-dotenv==0.14.0
+regex==2020.7.14
```

Comment:
This also needs to be removed!

Reply:
But why?

Reply:
Where exactly do you see regex used though?

Reply:
In validating username and email.

Reply:
Well since, regex is a module, do you see anywhere `import regex`?

Reply:
@SanketDG regex is imported as re(short name) just like some packages are published with different names on pip but install with different name.

Reply:
I initially thought that too but:

```
>>> re.__file__
'/home/sanketdg/.pyenv/versions/3.7.7/lib/python3.7/re.py'
>>> regex.__file__
'/home/sanketdg/.pyenv/versions/mentorship-backend/lib/python3.7/site-packages/regex/__init__.py'
>>>
```

Reply:
Thanks for correcting me. Just learned something new. :smile: